src/components/Login/css/styles.css

```diff
@@ -71,4 +71,12 @@ video {
 #testImage {
     width: 200px;
     height: 250px;
+}
+
+.centered {
```

Comment:
Is this .centered used anywhere? I could find it in Login.js file. Its not a issue, was just see why it was added.

Reply:
Originally yes, but not anymore. I still currently plan to use it however.